infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServerCache.java

```diff
@@ -24,11 +26,12 @@
     private final LoadingCache<Long, ILogData> dataCache;
     private final StreamLog streamLog;
 
-    public LogUnitServerCache(LogUnitServerConfig config, StreamLog streamLog) {
+    public LogUnitServerCache(LogUnitServerConfig config, StreamLog streamLog, ExecutorService executorService) {
```

Comment:
you can keep this of type Executor(more generic) instead of ExecutorService

Reply:
Done.